platform/pensando/sonic-platform-modules-dpu/sonic_platform/proto/system.proto

```diff
@@ -0,0 +1,225 @@
+// AMD representative for a copy.
+//
+// You agree that you will not reverse engineer or decompile the Materials,
+// in whole or in part, except as allowed by applicable law.
+//
+// THE MATERIALS ARE DISTRIBUTED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OR
+// REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
+//
+
+
+//------------------------------------------------------------------------------
+
+//
+// protobuf specification for the top level system object
+//------------------------------------------------------------------------------
+
+syntax = "proto3";
+package pds;
+
+import "gogo.proto";
+import "google/protobuf/timestamp.proto";
+import "meta/meta.proto";
+import "types.proto";
+
+service SystemSvc {
+  rpc SystemGet (types.Empty) returns (SystemGetResponse) {}
+  rpc InventoryGet (types.Empty) returns (InventoryGetResponse) {}
+  rpc QueueCreditsGet (types.Empty) returns (QueueCreditsGetResponse) {}
+  rpc InterruptGet(InterruptGetRequest) returns (stream InterruptGetResponse) {}
+}
+
+message TemperatureStats {
+  uint32 Temperature       = 1; // current temperature
+  uint32 MaxTemperature    = 2; // max temperature recorded
+  uint32 MinTemperature    = 3; // min temperature recorded
+  double AvgTemperature    = 4; // avg temperature
+  double StandardDeviation = 5; // standard deviation
+}
+
+message SystemTemperature {
+  TemperatureStats DieTemperature   = 1;
+  TemperatureStats LocalTemperature = 2;
+  TemperatureStats HbmTemperature   = 3;
+}
+
+message SystemVoltage {
+  uint32 CoreVoltage = 1;
+  uint32 ArmVoltage  = 2;
+}
+
+// type of the system led values
+enum LedStatus {
+  LED_STATUS_NONE  = 0;
+  LED_STATUS_GREEN = 1;
+  LED_STATUS_AMBER = 2;
+  LED_STATUS_OFF   = 3;
+}
+
+// type of the system reset reason values
+enum ResetReason {
+  RESET_REASON_UNKNOWN          = 0;
+  RESET_REASON_SW_POWER_CYCLE   = 1;
+  RESET_REASON_VRD_FAULT        = 2;
+  RESET_REASON_THERM_TRIP       = 3;
+  RESET_REASON_HOST_POWER_CYCLE = 4;
+  RESET_REASON_AC_POWER_CYCLE   = 5;
+  RESET_REASON_REBOOT           = 6;
+  RESET_REASON_PANIC            = 7;
+  RESET_REASON_PCIE_RESET       = 8;
+  RESET_REASON_ESECURE_ENABLED  = 9;
+  RESET_REASON_ASIC_WARM_RESET  = 10;
+}
+
+message PowerStats {
+  uint32 Power             = 1; // current power
+  uint32 MaxPower          = 2; // max power recorded
+  uint32 MinPower          = 3; // min power recorded
+  double AvgPower          = 4; // avg power
+  double StandardDeviation = 5; // standard deviation
+}
+
+message SystemPower {
+  PowerStats Pin   = 1;
+  PowerStats Pout1 = 2;
+  PowerStats Pout2 = 3;
+}
+
+message SystemFrequency {
+  uint32 ClockFrequency    = 1;
+  uint32 ArmClockFrequency = 2;
+  uint32 P4StageFrequency  = 3;
+}
+
+message SystemCurrent {
+  repeated uint32 Current = 1; // list of current sensor values in milliampere
+}
+
+// pmbus sensor status for current, voltage and temperature
+message PMBusSensorStatus {
```

Comment:
@SahilChaudhari @shanshri can we use and integrate with existing `sensors` on sonic?

````
root@sensors:~# sensors --help
Usage: sensors [OPTION]... [CHIP]...
  -c, --config-file     Specify a config file
  -h, --help            Display this help text
  -s, --set             Execute `set' statements (root only)
  -f, --fahrenheit      Show temperatures in degrees fahrenheit
  -A, --no-adapter      Do not show adapter for each chip
      --bus-list        Generate bus statements for sensors.conf
  -u                    Raw output
  -j                    Json output
  -v, --version         Display the program version

Use `-' after `-c' to read the config file from stdin.
If no chips are specified, all chip info will be printed.
Example chip names:
        lm78-i2c-0-2d   *-i2c-0-2d
        lm78-i2c-0-*    *-i2c-0-*
        lm78-i2c-*-2d   *-i2c-*-2d
        lm78-i2c-*-*    *-i2c-*-*
        lm78-isa-0290   *-isa-0290
        lm78-isa-*      *-isa-*
        lm78-*
```

Reply:
@prgeor, will consider this. Need to check work needed and ETA. Putting this as phase-II action item